chapters/cli/exit.rst

```diff
@@ -1,8 +1,25 @@
+====================
 Gracefully Exit Jina
 ====================
+---------------------------------------------------------------------------------
+Summary: In this section you will learn the ways to shut down Flow and exit Jina.
+---------------------------------------------------------------------------------
+
+Feature description and expected outcome
+----------------------------------------
+Jina provide several ways including Python and console to help you exit gracefully.
```

Comment:
I think we can shorten this to: 

There are several methods to exit Jina gracefully. 

Or do you think we are losing important information?

Reply:
Thanks! ✔️